.travis.yml

```diff
@@ -13,28 +13,63 @@ matrix:
   - python: 2.7
     env: CONDA_ENV=py27-min
   - python: 2.7
+    env: CONDA_ENV=py27-min SKIP_CI=--skip-optional-ci
```

Comment:
Actually, we can just run flakey tests for all tests allowed to fail and skip optional CI for all tests that must pass.  This avoids having new tests and should be more economical.